pom.xml

```diff
@@ -101,6 +118,14 @@
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>findbugs-maven-plugin</artifactId>
+          <configuration>
+            <maxRank>14</maxRank>
+            <includeTests>true</includeTests>
+          </configuration>
```

Comment:
Does findbugs activate with every build because of this? I don't see that happening?

Reply:
@ge0ffrey No it doesn't. Findbugs plugin execution is part of `fullProfile` defined in `kie-parent` and can be activated by `-Dfull` property. The point of this change is to tweak Findbugs configuration for this project. Without setting maxRank to this specific value, it doesn't any of the problems that this PR fixes.